docs/resources/scripts/linux_binary_installation.bash

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+##LINUX_BINARY_LOCALE
+locale  # check for UTF-8
+
+sudo apt update && sudo apt install -y locales
+# Any UTF-8 locale will work. Using en_US as an example
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+export LANG=en_US.UTF-8
```

Comment:
Wouldn't this overwrite the user's current locale? Maybe the script should skip this block if a UTF-8 locale is detected.

Reply:
Done in e9d392d